src/index.html

```diff
@@ -22,15 +22,15 @@
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <link rel="stylesheet" href="../base1/patternfly.css">
+    <link rel="stylesheet" href="patternfly-4-base.css">
```

Comment:
Still broken, I'm afraid :crying_cat_face: But my dist/ folder doesn't actually have that file:
```
❱❱❱ ls -l dist/
total 1248
-rw-rw-r--. 1 martin martin   19071 27. Mai 16:03 index.css
-rw-rw-r--. 1 martin martin   27453 27. Mai 16:03 index.css.map
-rw-rw-r--. 1 martin martin    1284 27. Mai 16:03 index.html
-rw-rw-r--. 1 martin martin   96367 27. Mai 16:03 index.min.js.gz
-rw-rw-r--. 1 martin martin 1114220 27. Mai 16:03 index.min.js.map
-rw-rw-r--. 1 martin martin     162 27. Mai 16:03 manifest.json
-rw-rw-r--. 1 martin martin     512 27. Mai 16:03 po.de.js
```

Is it supposed to be a disted file, or does that get built into the webpack?